build/dockerfiles/content_sets_centos8_appstream.repo

```diff
@@ -0,0 +1,5 @@
+[AppStream]
```

Comment:
ya'll should just switch to UBI and reuse what I already have downstream for this.

Reply:
It [is UBI](https://github.com/eclipse/che-devfile-registry/blob/master/build/dockerfiles/rhel.Dockerfile#L15), but none of the UBI repos contain skopeo AFAICT.

Reply:
well, the Brew internal ones do, such as **REDACTED**/content/dist/rhel8/8/x86_64/appstream/os/Packages/s/

Reply:
Sounds like *someone* is going to be using a custom `.repo` in the Brew build :wink:

Reply:
Nope, in Brew we don't need no steenkin' repos. Brew handles that for us. (Because it's locked down to only approved repos & rpms.)